intents/oversights/regression_to_mean.py

```diff
@@ -0,0 +1,293 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""This module detects the cases of regression to the mean in the top-k
+intent.
+Regression toward the mean is the phenomenon that arises if a random
+variable is extreme on its first or first few measurements but closer
+to the mean or average on further measurements.
+If the top-k is queried for a time window in which the results look much
+different from the previous window, or the next window of the same size, the
+oversight suggestion is passed.
+"""
+
+import sys
+
+import datetime
+
+import math
+
+sys.path.append("../..")
```

Comment:
This is a little weird. Can we add __init__.py in our folders to make them packages, so that they automatically can be imported?